bme_test_suite.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- hosts: deploy_node
+  name: Install and run tests
+  vars:
+    tempest_root: /opt/tempest_untagged
+    test_name: persistent-resources-tests
```

Comment:
These are default values?
Why are the persistent-resources-tests the default ones?